### ISSUE TITLE
Make `yarn rw test` forward everything to "jest"

### ISSUE THREAD

**Issue by peterp** [redwoodjs/redwood#2708]
Right now we "hide" that you're running jest when you run `yarn rw test`

We do this because we set some helpful defaults for you. I think we should still set those helpful defaults, but we should just make `rw test` an alias of `jest`.

For inspiration we can look at how we handle the prisma command.

So that running:
`yarn rw test --help` will output jest's help.

Additionally we can remove "code-coverage" and "watch" options from our own code.

**Comment by thedavidprice** [comment]
@peterp do you think this will involve a breaking change? I'm assuming "no" but want to confirm with you before prioritizing.

**Comment by KrisCoulson** [comment]
I think I found a good use case for this. Dunno if it is possible to make `yarn jest` runnable from root of a redwood app. @peterp @thedavidprice  


 @dac09 recently got a bunch of redwood devs trying wallabyjs plugin for vscode/ jest testing for better testing DX.  When trying to run the plugin there are some issues. For example `scenario` is not defined.
 
![image](https://user-images.githubusercontent.com/6943688/134790859-0779127a-e390-467c-8876-ce8a4596f324.png)

I reached out to their support it seems that wallaby has a hard time with redwood by default because we essentially can't run `yarn jest` from the root of a redwood app.
 
![image](https://user-images.githubusercontent.com/6943688/134599158-dd607a37-780e-4342-9fa6-005f771b761e.png)


When running `yarn jest` from our redwood app root we get the same issue
![image](https://user-images.githubusercontent.com/6943688/134791039-c4c2648d-5fd9-4132-9a08-b9e7ff6fe0fa.png)

To get around this one of their team members suggest adding a `jest.config.js` to the root of  a redwood app with the config below. Where we export `projects` from jest config [projects array](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig). When adding this the app was able to run 'yarn jest` and everything ran smoothly.
```javascript 
if (!process.env.DATABASE_URL) {
  const process = require('process')
  const path = require('path')
  const cacheDirDb = `file:${path.join(__dirname, '.redwood', 'test.db')}`
  process.env.DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb

  const execa = require('execa')
  execa.sync(
    `yarn rw`,
    ['prisma db push', '--force-reset', '--accept-data-loss'],
    {
      cwd: path.join(__dirname, 'api'),
      stdio: 'inherit',
      shell: true,
      env: {
        DATABASE_URL: process.env.DATABASE_URL,
      },
    }
  )
}

module.exports = {
  projects: ['<rootDir>/api/', '<rootDir>/web/'],
}
```

I am wondering If we pass everything through `jest` if that would also allow us to just run `yarn jest` from root alongside  of `yarn rw test`


 Here is a link to the conversation in the wallbyjs support discord the thread inside has some really good context to given that some of the redwood service test run really slow.
 https://discord.com/channels/694023081168994355/694035930541064293/889410850715226162

**Comment by dac09** [comment]
OK.... so I've been playing around with this today, and I think the correct configuration is this (atleast for Wallaby)

```
//./jest.config.js
require('dotenv-defaults/config')

module.exports = {
  projects: ['<rootDir>/api/', '<rootDir>/web/'],
}
```

I'll double check whether this fixes the other issues we've been seeing as well and create a PR if it does.

**Comment by thedavidprice** [comment]
This will be resolved in #4096
